performance-test/README.md

```diff
@@ -0,0 +1,23 @@
+## Steps to run an upload test with neqo-client and neqo-server:
+
+1. Build the release version of neqo-client and neqo-server by running
+   `cargo build --release`
+1. Start neqo-server. `./target/release/neqo-server --db ./test-fixture/db`
+1. Start neqo-client and specify parameters to start the upload test.
+   ` ./target/release/neqo-client http://127.0.0.1:4433/  --test upload  --upload-size ${size_in_bytes}`
+
+## To enable log messages for analyzing upload performance
+
+This can be done by setting the `RUST_LOG` environment variable to `neqo_transport=info`.
+For example, the command below starts neqo-client and uploads 8MB of content to the server.
+```
+RUST_LOG=neqo_transport=info ./target/release/neqo-client http://127.0.0.1:4433/ --test upload --upload-size 8388608 &>upload.log
+```
```

Comment:
What is the performance impact of enabling logging? Also, is a debug build OK here or should it be an optimized release build?

Reply:
Ideally, we'd want to take a packet trace and SSLKEYLOG during a performance run, decrypt the trace via Wireshark, conveft it to - say - JSON, and extract the information to plot out of *that*, rather than slowing down the endpoints by logging there.

Reply:
Wireshark will show you what on the wire :-) What the logs show depend on in which order neqo pushes and pulls data from the sockets, which depends on its internal scheduling logic.